frontend/src/testing/data/transitionData.ts

```diff
@@ -5,17 +5,31 @@ export default createTransitions([
   ['43-4051', 'Customer service representatives', 0.0421, 17.9, 37300],
   ['11-9051', 'Food service managers', 0.0407, 28.76, 59820],
   ['35-3011', 'Bartenders', 0.039, 13.46, 28000],
-  ['43-6014', 'Secretaries and administrative assistants, except legal, edical, and executive', 0.0391, 18.84, 39180,],
+  [
+    '43-6014',
+    'Secretaries and administrative assistants, except legal, edical, and executive',
+    0.0391,
+    18.84,
+    39180,
+  ],
   ['43-9061', 'Office clerks, general', 0.034, 17.48, 36360],
   ['41-2011', 'Cashiers', 0.0312, 11.72, 24370],
   ['11-9000', 'Other Management Occupations', 0.0303, 49.31, 102560],
-  ['35-1012', 'First-line supervisors of food preparation and erving workers', 0.0231, 17.77, 36960,],
-  ['41-1011', 'First-line supervisors of retail sales workers', 0.0199, 22.03, 45830,],
+  [
+    '35-1012',
+    'First-line supervisors of food preparation and erving workers',
```

Comment:
should "erving" be "serving?"

Reply:
this is fixed in the latest version. Are we all set?